site/source/tutorials/008_animations/index.md

```diff
@@ -0,0 +1,370 @@
+
+You also need to be familiar with TypeScript as Dojo 2 uses it extensively. For more information, refer to the [TypeScript and Dojo 2](../../docs/fundamentals/typescript_and_dojo_2/) article.
+
+{% section %}
+
+## What is the web animations API
+
+The [Web Animations API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Animations_API) provides programatic control over web animations via the timing model and the animation model. This allows animations to be created and controlled via javascript with access to playbackrate, iterations, events etc... something which would previously have required the use of `requestAnimationFrame` or the dreaded `setInterval`. Dojo 2 provides a meta that can apply Web Animations to the rendered vdom in the widgets you create. This allows properties such as `play` and `duration` to be reactive to state changes and fits in neatly within the Dojo 2 eco stystem.
+
+## Introducing the zombies
+
+Looking at the `Zombies.ts` file within the initial project we can see that we have a series of divs representing two zombies, each with a body and two legs. Via the magic of css these turn into two zombies when you open your browser. To demonstrate the use of Web Animations with Dojo 2 we make the zombies walk towards each other when they are clicked.
+
+{% aside 'Meta' %}
+Dojo 2 meta provides a means to get and set properties against the generated HTML without exposing the domNode itself.
+{% endaside %}
+
+{% task 'Animating our zombies' %}
+
+To animate the zombies, we use the `@dojo/widget-core/meta/WebAnimation` meta. This takes a `key` and `AnimationProperties`.
+Lets add animation properties to our first Zombie.
+
+{% instruction 'Add the following to the render function in Zombies.ts' %}
+
+``` typescript
+// add the import
+import WebAnimation from '@dojo/widget-core/meta/WebAnimation';
+
+//add to the top of the render function
+const zombieOneMoveAnimation = {
+	id: 'zombieOneMove,
+	effects: [
+		{ left: '0%' },
+		{ left: '35%' }
+	],
+	timing: {
+		duration: 2000,
+		easing: 'ease-in',
+		fill: 'forwards'
+	},
+	controls: {
+		play: true
+	}
+};
+
+this.meta(WebAnimation).animate('zombieOne', zombieOneMoveAnimation);
+```
+
+{% aside 'Reminder' %}
+If you cannot see the zombies, make sure you have run `dojo build -w` to build the application and start the development server.
+{% endaside %}
+
+Refresh your browser and you should now see the left zombie moving across the screen for 2 seconds.
+
+So let's talk about the animation properties we've created here:
+
+- A unique id for this animation
+- Effects array depicts the steps for the animation, in this case we're animating the `left` style from `0%` to `35%`
+- The timing object specifies the duration and easing effects for the animation. The fill param specifies that the animation should finish in it's final position
+- The controls object in this case simply sets play to `true`
+
+{% task 'Animating our zombie on click' %}
```

Comment:
Do we prefer on click or `onclick` ... change if you feel it should correlate to the event rather than as words.